app/src/utils/Utils.ts

```diff
@@ -252,3 +251,30 @@ export const getFormattedIdentitySource = (identitySource: SYSTEM_IDENTITY_SOURC
       return null;
   }
 };
+declare global {
```

Comment:
Is there any way we could avoid declaring this in the global scope and just import this as a util function on a case-by-case basis? I'm not super well versed on the pitfalls of changing `Array.prototype`, but I think it's generally advised to avoid it (https://eslint.org/docs/latest/rules/no-extend-native); Unless the code needs to run this as an array protoype method? 🤔

Reply:
it doesn't need to extend the array class, I wanted to be able to chain the methods together so you could for example `[].alphabetizeObjects().map()`. 

An example is in `ProjectParticipantsPage.tsx`, I think it keeps it a little cleaner with the react code but am happy to change that

Reply:
turned this into a regular function that accepts a generic array of objects and sorts on a given property